news/2025/2025-04-05-new-featured-artist-lexycat.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "New Featured Artist: lexycat"
+date: 2025-04-05 21:00:00 +0000
+---
+
+**lexycat** crawls out from the depths of the early 2000s internet landscape and into our Featured Artist library!
+
+![](https://assets.ppy.sh/artists/468/header.jpg)
+
+Like many osu! players, self-proclaimed cybernetic angel **lexycat** grew up on the internet. Surrounded by digital hearts, zip files, YTPMVs, kitty cats, nightcore girls, dubstep, and VOCALOID, **lexycat** creates music reflective of a online landscape that anyone in the e-age of olde is more than familiar with.
+
+A total of **10** unique tracks are part of [**lexycat**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/468). Pick one of the pre-timed `.osz`s for your next map. Or pick all 10. We won't stop you.
+
+Sample some of **lexycat**'s tunes down here:
+
+### [lexycat - incision](https://assets.ppy.sh/artists/468/crystal%20forest/lexycat%20-%20incision.osz)
+
+[COE](/wiki/Community/cavoe's_osu!_event) 2025 is on the horizon, but memories of last year's event are as fresh as yesterday... Or as fresh as right now, if you recall [mangomizer](https://osu.ppy.sh/users/1893718)'s thumbs tapping to [HEAVENLY MOON](https://osu.ppy.sh/users/13681283)'s map of [incision](https://osu.ppy.sh/beatmapsets/2208878):
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/468/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### [lexycat - glitter *](https://assets.ppy.sh/artists/468/glitter/lexycat%20-%20glitter%20.osz)
+
+Use your own thumbs on [this osu!taiko map](https://osu.ppy.sh/beatmapsets/2300144) hosted by [Irone OSU](https://osu.ppy.sh/users/10678230)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/468/glitter/lexycat%20-%20glitter%20.mp3">
+</audio>
+
+### [lexycat - Abyssal Storm](https://assets.ppy.sh/artists/468/Abyssal%20Storm/lexycat%20-%20Abyssal%20Storm.osz)
+
+Or stretch the rest of your fingers on [this osu!mania map](https://osu.ppy.sh/beatmapsets/2285560) hosted by [Usagi_](https://osu.ppy.sh/users/16046205)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/468/Abyssal%20Storm/lexycat%20-%20Abyssal%20Storm.mp3">
+</audio>
+
+---
+
+[**lexycat**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/468) has even more tracks for you to peruse. Do yourself a favor by reading song titles on her listing instead of reading this news post.
```

Comment:
```suggestion
[**lexycat**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/468) has even more tracks for you to peruse. Do yourself a favour by reading song titles on her listing instead of reading this news post.
```